extension/content/personas.css

```diff
@@ -90,6 +95,20 @@
   font-weight: bold !important;
 }
 
+#personas-toolbar-button.seamonkey, #personas-toolbar-button.seamonkey:hover, #personas-toolbar-button.seamonkey:hover:active {
```

Comment:
yes they are necessary. If not, other hover and active rules win and the toolbar icon is hidden.